ssc/CMakeLists.txt

```diff
@@ -200,6 +200,7 @@ if(UNIX AND NOT SAMAPI_EXPORT)
 	add_custom_command(
 		TARGET ssc
 		POST_BUILD
+		COMMAND codesign --force --deep --sign - $<TARGET_FILE:ssc>
```

Comment:
This is causing me trouble on the Linux build - getting "codesign - not found" Is there a new dependency that should be added to the wiki?

Reply:
Fixed in 8c9657eb7dab4d01be4c8faab7bd46ab96bfc4ce